src/Exception/MediaTypeNotAssociatedWithMpxException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Drupal\media_mpx\Exception;
+
+/**
+ * Class MediaTypeNotAssociatedWithMpxException.
+ *
+ * @package Drupal\media_mpx\Exception
+ */
+class MediaTypeNotAssociatedWithMpxException extends \Exception {}
```

Comment:
Same question, should this extend some other exception type? At a glance it almost feels like `\LogicException`, or perhaps `\InvalidArgumentException`.

Reply:
Moved to \Logic one. Great call :)